### ISSUE TITLE
Language selection improvement for `gost-r-7-0-5-2008-numeric.csl`

### ISSUE THREAD

**Issue by e-khurmamatov** [citation-style-language/styles#6532]
This is a follow-up to the previous pull request and this comment https://github.com/citation-style-language/styles/pull/6522#issuecomment-1540010299

I would like to request an improvement for the `gost-r-7-0-5-2008-numeric.csl` style. Currently, the `default-locale` field is set to `en-US`, which means that the style is in English by default. However, the style also supports Russian (`ru-RU`) as a language option.

It would be helpful to make the language selection more accessible through the Zotero interface. Currently, the language selection is grayed out and not easily accessible. It would be great to have a more user-friendly way to switch the language of the style.

**Comment by adam3smith** [comment]
Yeah, that line got put in there on purpose (it's even mentioned in the summary), but I don't see a reason it should be -- we'll take it out.

**Comment by POBrien333** [comment]
Closed with #7743
